resources/css/forms.css

```diff
@@ -1,104 +1,104 @@
-textarea,
-select {
+.standard input[type='text'],
+.standard input[type='password'],
+.standard input[type='email'],
+.standard textarea,
+.standard select {
     @apply rounded border-2 p-3 w-full;
 }
 
-select {
+.standard select {
     @apply appearance-none;
 }
 
-input.nav-search {
+.standard input.nav-search {
     @apply pl-10 pr-3 py-2;
 }
 
-input[type='checkbox'] {
+.standard input[type='checkbox'] {
     @apply mr-1;
 }
 
-.help-block {
+.standard .help-block {
     @apply text-red-500 text-sm;
 }
 
 /** Markdown editor **/
-.form-group .editor {
+.standard .form-group .editor {
     @apply rounded-t-none;
 }
 
-.form-group .editor-toolbar {
+.standard .form-group .editor-toolbar {
     @apply border-2 border-b-0 rounded rounded-b-none border-gray-300;
 }
 
-.form-group .CodeMirror {
+.standard .form-group .CodeMirror {
     @apply border-2 rounded rounded-t-none border-gray-300;
 }
 
 /** Choices.js **/
-.choices__input.choices__input--cloned {
+.standard .choices__input.choices__input--cloned {
     @apply hidden;
 }
 
-.choices__inner {
+.standard .choices__inner {
     @apply rounded border-2 border-gray-300 w-full bg-white h-12 p-2 flex items-center !important;
 }
 
-.choices__list--dropdown {
+.standard .choices__list--dropdown {
     @apply border-2 border-gray-300 !important;
 }
 
-.choices__list--single {
+.standard .choices__list--single {
     @apply p-0 !important;
 }
 
-.choices__item {
+.standard .choices__item {
     @apply text-sm bg-gray-200 text-gray-700 cursor-pointer rounded px-2 py-1 my-0 border-0 !important;
 }
 
-.choices__item.choices__item--choice {
+.standard .choices__item.choices__item--choice {
     @apply rounded-none py-2 !important;
 }
 
-.choices__item.choices__item--choice.choices__item--selectable {
+.standard .choices__item.choices__item--choice.choices__item--selectable {
     @apply bg-white rounded-none text-gray-700 p-2 !important;
 }
 
-.choices__item.choices__item--choice.choices__item--selectable:hover {
+.standard .choices__item.choices__item--choice.choices__item--selectable:hover {
     @apply bg-gray-200 !important;
 }
 
-.choices__item.choices__item--selectable {
+.standard .choices__item.choices__item--selectable {
     @apply bg-lio-500 border-lio-500 text-white !important;
 }
 
-.choices__list--single > .choices__item.choices__item--selectable {
+.standard .choices__list--single > .choices__item.choices__item--selectable {
     @apply bg-white text-gray-700 !important;
 }
 
-.choices__item.is-highlighted {
+.standard .choices__item.is-highlighted {
     @apply bg-lio-600 border-lio-600 text-white !important;
 }
 
-.choices[data-type*='select-one']:after {
+.standard .choices[data-type*='select-one']:after {
     border-color: theme('colors.gray.500') transparent transparent transparent !important;
 }
 
-.choices[data-type*='select-one'] .choices__input {
+.standard .choices[data-type*='select-one'] .choices__input {
     @apply rounded-none border-b-2 border-gray-300 !important;
 }
```

Comment:
Could have just wrapped the place where the file is imported in a `.standard` class heh.